### ISSUE TITLE
Barbed crescent not obtainable

### ISSUE THREAD

**Issue by BlessedXxX** [LegionDark/Issues#445]
me and like 4 to 5 other blue mages ahve tried to gain barbed crescent and it just will not let us learn would like to see if a Gm can obtain just to see if its being a pain or if its really broken

**Comment by TeoTwawki** [comment]
This has been brought up like a million times - its not learnable in the normal fashion but has been added as a scripted "fake drop" to specific mobs. Once the server catches up to DarkStar it will learnable in retail fashion.

**Comment by TeoTwawki** [comment]
If you want this spell, camp the shadow NMs in Fei'Yin for now.

**Comment by TeoTwawki** [comment]
Funny thing about includes in lua script (know as "require"): if you load one as a global it stays in memory even if your current file is missing it, so you don't get an error about it being missing from a script it should be in. Like one spell is missing the global "magic.lua" but you have already cast one that wasn't - both spells function normally. But if you just logged in and cast the one that had it missing, there's an error that miraculously only you got while everyone else has it working..

So here we are, ever cranking up a rate of a spell until its freakin 40% with in game GMs wondering HOW people are not so unlucky because when we test we get the spell just fine..

Why hath RNGesus forsaken thee? Coz somebody (cough Loki cough :smile: ) forgot to include status.lua so the script could know wtf JOB_BLU even is. I had closed this because it'd been reported and a GM confirmed they were able to get the spell previously. I just looked to see what the % was set to and noted the missing require. This happens because he has to much to do and quick copy-pastes his header area into 50 odd files in a hurry, which is where require lines sit.

Try it now. at 40% rate for anyone who's mainjob is blu there is no way you won't get it really fast.
